src/PrestaShopBundle/Controller/Api/StockController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * 2007-2016 PrestaShop
```

Comment:
The licenses are outdated. Time to execute `app/console prestashop:licences:update` :smile: